resources/definitions/atom2.def.json

```diff
@@ -0,0 +1,63 @@
+{
+  "name": "Atom 2",
+  "version": 2,
+  "inherits": "fdmprinter",
+  "metadata": {
+    "visible": true,
+    "author": "Victor (Yu Chieh) Lin",
+    "manufacturer": "Layer One",
+    "file_formats": "text/x-gcode",
+    "platform_offset": [0,0,0],
+    "machine_extruder_trains":  { "0": "atom2_extruder_0"
+    }
+  },
+
+  "overrides": {
+    "machine_name":           { "default_value": "Atom 2" },
+    "machine_shape":          { "default_value": "elliptic" },
+    "machine_width":          { "default_value": 210 },
+    "machine_depth":          { "default_value": 210 },
```

Comment:
I can't find a page where this printer is actually sold with specs (the [website](https://atom3dp.squarespace.com/) is giving 404 errors and [this store](https://www.atom3dp.com/en/store/) doesn't list it), but according to [this review](https://all3dp.com/stealth-black-layer-one-atom-2-0-delta-3d-printer-review/) the printer is 22x32cm. Was this a typo, or was it intentional that you're 10mm short of the published dimensions?

Reply:
Atom 2.0 has been phased out by Layer One, so it's hard to find official data, but here's their crowdfunding [page](https://www.zeczec.com/projects/atom2), there's spec [photo](https://farm8.staticflickr.com/7533/15704847653_dbb9e03d1b_o.jpg) about three quarter way down the page.
_Even tho it's officially phased out, there's still a large user base in Taiwan that's on Atom 2.0_
I'm intentionally short 10mm, the bed of the Atom 2.0 is held down with "clips", these clips extend into the build area. When printing at the edges, it's possible that the head hits these clips and cause problems.

Reply:
Okay, as long as it's intentional that is okay!